Ssmpu/Memory_Protection_Unit.adoc

```diff
@@ -57,13 +60,29 @@ The A field in an MPU entry’s configuration register encodes the address-match
 It is the same as PMP/ePMP.
 
 === Encoding of Permissions
-MPU has three kinds of regions: U-mode region, S-mode region, and shared regions. Normally, when the S bit is set, the region could be either a S-mode region or a shared region. When the S bit is clear, the region could be either a U-mode region or a shared region.
 
-S-mode regions indicate that the R/W/X permissions are enforced on S-mode accesses. M-mode accesses are not affected and U-mode access will trigger faults. 
-For U-mode regions, any S-mode access matching the MPU entry will trigger faults; the R/W/X permissions apply only to U modes.
-For shared regions, the permissions are enforced for both U and S modes, according to the table below.
 
-S-mode can always modify the value in S bit.
+MPU has three kinds of rules: *U-mode-only*, *S-mode-only*, and *Shared-Region* rules.
+The S bit marks a rule as *S-mode-only* when set and *U-mode-only* when unset.
+The encoding ``mpucfg.SRWX=1000`` is reserved and the encoding ``mpucfg.RW=01`` and the encoding ``mpucfg.SRWX=1111`` encodes a Shared-Region.
```

Comment:
let's clean this up a bit, let's say something like "mpucfg.RW=01  encodes a shared region and mpucfg.SRWX=1000 is reserved for future standard use". mpucfg.SRWX=1111 is mentioned later on, and since it's a special case let's leave it there.